client/components/Editor/views/ReferenceView.ts

```diff
@@ -4,19 +4,21 @@ import { EditorView } from 'prosemirror-view';
 export default class ReferenceView {
 	view: EditorView;
 	dom: HTMLAnchorElement;
-	contentDOM: HTMLAnchorElement;
 
 	constructor(node: Node, view: EditorView) {
 		const { targetId, label } = node.attrs;
 
 		this.view = view;
-		this.dom = this.contentDOM = document.createElement('a');
+		this.dom = document.createElement('a');
 
 		this.dom.textContent = label || `(ref?)`;
 		this.dom.classList.add('reference');
 		this.dom.setAttribute('data-node-type', 'reference');
 		this.dom.setAttribute('data-target-id', targetId);
-		this.dom.setAttribute('href', `#${targetId}`);
+
+		if (!this.view.editable) {
+			this.dom.setAttribute('href', `#${targetId}`);
+		}
```

Comment:
@idreyn After removing the contentDOM property from the node view, the click-dropdown behavior of references in the (editable) editor stopped working. This seemed to work fine, but now we have anchor tags without `href`s, which somewhat defeats the purpose of moving this logic into a node view. Do you have any thoughts?